<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="information">
      <property name="maximumSize">
       <size>
        <width>359</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#information{
	width: 360px;
	height: 60px;
	border-radius: 9px;
	background-color: 'white';
	margin-left:56%
}
#information:hover {
	background-color: #F8F8FF;
}</string>
      </property>
      <property name="text">
       <string>Information</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="chat">
      <property name="maximumSize">
       <size>
        <width>359</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#chat{
	width: 360px;
	height: 60px;
	border-radius: 9px;
	background-color: 'white';
	margin-bottom: 16px;
	margin-left: 56%;
}
#chat:hover {
	background-color: #F8F8FF;
}</string>
      </property>
      <property name="text">
       <string>Chat</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="staff_login">
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#staff_login{
	width: 260px;
	height: 36px;
	border-radius: 9px;
	background-color: 'white';
	margin-right: 16%
}
#staff_login:hover {
	background-color: #F8F8FF;
}</string>
      </property>
      <property name="text">
       <string>Staff Login</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignTop">
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">margin-left: 16%</string>
      </property>
      <property name="text">
       <string>SeeComm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
